doc/reference/storages.rst

```diff
@@ -144,31 +144,46 @@ Options:
 Noteworthy non-included storages
 ================================
 
-A number of important ZODB storages are distributed separately, including:
+A number of important ZODB storages are distributed separately.
+
+Base storages
+-------------
+
+Unlike the included storages, all the implementations listed in this section
+allow multiple processes to share the same database.
+
+NEO
+  `NEO <https://lab.nexedi.com/nexedi/neoppod>`_ can spread data among several
+  computers for load-balancing and multi-master replication. It also supports
+  asynchronous replication to off-site NEO databases for further disaster
+  resistance without affecting local operation latency.
```

Comment:
I'm not sure it's the place to start listing drawbacks of each solution. The main limitations (like linux-only but also no blob support) look visible enough on the linked page.

The use of `epoll()` is the only reason why it requires Linux. In 2 places:
  https://lab.nexedi.com/nexedi/neoppod/blob/master/neo/lib/event.py
  https://lab.nexedi.com/nexedi/neoppod/blob/master/neo/tests/threaded/__init__.py

On our side, it's the contrary: we only develop and run test suites on Linux. We're open to contributions but unless someone provides test bots for other OS, we could only maintain it if the alternative implementation is not OS-specific, like select or libevent.

I'll see with @vpelletier